x/examples/website-wrapper-app/wrapper_app_project/.scripts/build.mjs

```diff
@@ -94,11 +100,28 @@ export default async function main(
       additionalDomains = [additionalDomains];
     }
 
+    const [entryTld, ...entryDomainParts] = entryDomain.split(".").reverse();
+
+    // Infer an app name from the base entry domain part 
+    const defaultAppName = entryDomainParts[0].toLocaleLowerCase().replaceAll(
+      /\w[a-z0-9]*[-_]*/g,
+      match => {
+        match = match.replace(/[-_]+/, " ");
+
+        return match.charAt(0).toUpperCase() + match.slice(1).toLowerCase();
+      }
+    )
+  
+    // Infer an app ID from the entry domain by reversing it (e.g. `www.example.com` becomes `com.example.www`)
+    const defaultAppId = entryTld + "." + entryDomainParts.join(".").replaceAll("-", "");
+
     fs.writeFileSync(
       destinationFilepath.replace(/\.handlebars$/, ""),
       template({
         platform,
         entryDomain,
+        appId: appId ?? defaultAppId,
```

Comment:
Is `appId` required to be lower case? If so, we need to make sure both are lower case here.